### ISSUE TITLE
Windjammers 2 - CUSA15634

### ISSUE THREAD

**Issue by eniols** [red-prig/fpps4-game-compatibility#578]
# Game name
Windjammers 2

## Code
CUSA15634

## Used emulator's version
red-prig/fpPS4@5760dfc

## Current status
Nothing

## Error
```
nop nid:libSceHttp:0CAF86A1708D4F4E:sceHttpsLoadCert
``` 

## Description
Emulation halts after nopping a service.

## Log
[CUSA15634 - LOG.txt](https://github.com/red-prig/fpps4-game-compatibility/files/10273880/CUSA15634.-.LOG.txt)

**Comment by GamingDuckingalt** [comment]
Retest

**Comment by Kalorte** [comment]
Retested with
https://github.com/red-prig/fpPS4/commit/184368fb29b169194aa2a67fda614736ecc5c7d6

No error but very long to load ... I don't know how much time
So it "stuck" on a black screen with around 50 FPS

Last log:
[main:15372] open:/app0/shaders/vs_2d.sb 0 (000)
[main:15372] open:/app0/shaders/fs_circle_mask.sb 0 (000)
RB_CompileShader
[main:15372] open:/app0/shaders/vs_2d.sb 0 (000)
[main:15372] open:/app0/shaders/fs_ring.sb 0 (000)
RB_CompileShader
[FMOD Studio sample load thread:7192] open:/app0/data/sets/MasterBank.bank 0 (000)
[main:15372] open:/app0/data/sets/clut/clut_discshade.img 0 (000)
[FMOD Studio update thread:17544] pthread_attr_init
[FMOD Studio update thread:17544] pthread_create:0000000020072D20 FMOD nonblocking thread (0)
[FMOD Studio update thread:17544] pthread_attr_destroy
[FMOD nonblocking thread (0):22360] BeginThread:0000000020072D20
[FMOD nonblocking thread (0):22360] open:/app0/data/sets/MasterBank.bank 0 (000)
[TWin32WidgetSet.Destroy] WARNING: There are 1 TimerInfo structures left, I'll free them

**Comment by h3ckD0g** [comment]
Challenge accepted ;)

**Comment by SonicThePS4** [comment]
It seems like we're getting somewhere. It really boots up! Only downside, is that there is no audio. But you can get through a main menu screen. Just that you can't see what are you going or what mode is highlighting. But when you manage to get In-Game, that's when it'll kick you back out.

![image](https://user-images.githubusercontent.com/39503785/213047020-8871c479-6647-403e-9a7b-dca024b8f2d9.png)

**Comment by gandalfthewhite19890404** [comment]
Any errors?

**Comment by SonicThePS4** [comment]
> Any errors?

Nope, nu-uh. Far as i can tell, it's runs pretty straight forward, until you get to the game play. ^^
